reflekt-plugin/src/main/kotlin/io/reflekt/plugin/analysis/processor/instances/ClassOrObjectInstancesProcessor.kt

```diff
@@ -1,21 +1,19 @@
 package io.reflekt.plugin.analysis.processor.instances
 
+import io.reflekt.plugin.analysis.processor.*
 import io.reflekt.plugin.analysis.processor.isPublicNotAbstractClass
 import io.reflekt.plugin.analysis.processor.isPublicObject
-import org.jetbrains.kotlin.psi.KtClass
-import org.jetbrains.kotlin.psi.KtClassOrObject
-import org.jetbrains.kotlin.psi.KtElement
-import org.jetbrains.kotlin.psi.KtObjectDeclaration
+import org.jetbrains.kotlin.psi.*
 import org.jetbrains.kotlin.resolve.BindingContext
 
-open class ClassOrObjectInstancesProcessor<T: KtClassOrObject>(override val binding: BindingContext) : BaseInstancesProcessor<List<KtClassOrObject>>(binding) {
-    override val instances: MutableList<T> = ArrayList()
+open class ClassOrObjectInstancesProcessor<T : KtClassOrObject>(override val binding: BindingContext) : BaseInstancesProcessor<MutableList<T>>(binding) {
+    override val fileToInstances: HashMap<String, MutableList<T>> = HashMap()
```

Comment:
and maybe create a typealias for HashMap<String, MutableList<T>>? it's used everywhere